spec/features/cloudflare_turnstile_spec.rb

```diff
@@ -0,0 +1,121 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+RSpec.describe "Cloudflare Turnstile" do
+  before(:each) do
+    RailsCloudflareTurnstile.configure do |c|
+      c.mock_enabled = true
+      c.enabled = false
+    end
+  end
+
+  describe "User Registration Form" do
+    it "shows error when cloudflare validation fails", :js do
+      visit new_user_registration_path
+
+      fill_in "Email", with: "test@example.com"
+      fill_in "Password", with: "password123"
+      fill_in "Password confirmation", with: "password123"
+      
+      # Override the mocked value to simulate failure
+      page.execute_script("document.querySelector('input[name=\"cf-turnstile-response\"]').value = 'invalid';")
```

Comment:
Remove this every place where you added it. It's not necessary if we set `mock_enabled` to false.